CHANGELOG.md

```diff
@@ -13,9 +13,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Add example to `opentelemetry-instrumentation-openai-v2`
```

Comment:
I think we want to keep openai specific changes in it's [own CHANGELOG](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/instrumentation-genai/opentelemetry-instrumentation-openai-v2/CHANGELOG.md)

Reply:
cool. lemme fix that